pkg/metrics/providers/graphite.go

```diff
@@ -109,7 +110,7 @@ func (g *GraphiteProvider) RunQuery(query string) (graphiteResponse, error) {
 // IsOnline runs a simple Graphite query and returns an error if the
 // API is unreachable.
 func (g *GraphiteProvider) IsOnline() (bool, error) {
-	_, err := g.RunQuery("")
+	_, err := g.RunQuery("target=test")
```

Comment:
I expect this _may_ stop working once we mature the `RunQuery` function, so we may need to revisit this down the road.